var/spack/repos/builtin/packages/py-onnx/package.py

```diff
@@ -28,7 +31,13 @@ class PyOnnx(PythonPackage):
     version("1.6.0", sha256="3b88c3fe521151651a0403c4d131cb2e0311bd28b753ef692020a432a81ce345")
     version("1.5.0", sha256="1a584a4ef62a6db178c257fffb06a9d8e61b41c0a80bfd8bcd8a253d72c4b0b4")
 
+    # pyproject.toml
+    depends_on("python@3.8:", type=("build", "run"), when="@1.15:")
+
     depends_on("py-setuptools", type="build")
+    # @61: in pyproject.toml when @1.15:, but upgraded to @64: due to editable_mode
```

Comment:
Might be worth submitting a PR to fix this bug upstream too

Reply:
https://github.com/onnx/onnx/pull/5827

Reply:
```suggestion
    # @61: in pyproject.toml when @1.15:, but upgraded to @64: due to editable_mode
    # see https://github.com/onnx/onnx/pull/5827
```